src/lib/download-raw.js

```diff
@@ -87,8 +91,9 @@ async function download_raw_publisher_feed(publisher_feed, callback) {
 
     /* Traverse all the pages available since our last run */
     while (true) {
-        /* Sleep - avoid hitting publisher's api too hard */
-        await Utils.sleep("oa-rpde-page-iter", 1);
+        if (Settings.sleepWhileDownloadRawSeconds > 0) {
```

Comment:
personally I wouldn't clutter the code checking if >0 as there is no harm in passing 0 seconds to the timer.

Reply:
Good point,  thanks